mina.netty/src/main/java/org/kaazing/mina/netty/config/InternalSystemProperty.java

```diff
@@ -35,7 +35,7 @@
 
     // A worker is serving multiple UDP child channels and they share an Agrona buffer.
     // This configures the buffer size in bytes. It must be a positive power of 2
```

Comment:
It doesn't have to be a positive power of 2, Agrona will round it up to the next highest power of 2 if it isn't. Change the comment to say "It will be rounded up to the next highest power of 2".